frontend/.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": true,
+  "bracketSpacing": true,
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "printWidth": 80,
```

Comment:
I'd change printWidth to at least 100 or even 120

Reply:
Fixed. Please check it out. Thanks.